src/app/clients/ActionBar.tsx

```diff
@@ -142,9 +142,9 @@ const ActionBar: React.FC<Props> = ({ selected, data }) => {
                 </MenuList>
             </Menu>
             <Menu
-                open={actionAnchorEl !== null}
-                onClose={() => setActionAnchorEl(null)}
```

Comment:
Try to fix the navBar taking up the whole screen on open and close :)

Reply:
I can't reproduce this on any browser under any viewport size...